tests/Compiler/Fixture/PhpContentExtractor/echo_variable_with_comments.blade.php

```diff
@@ -4,6 +4,6 @@
 }}
 -----
 <?php
-/** file: foo.blade.php, line: 1 */ echo e(/** @var  string $foo */
+/** file: foo.blade.php, line: 1 */ echo e(/** @var string $foo */
```

Comment:
This is a fixture for a test and should not be styled
```suggestion
/** file: foo.blade.php, line: 1 */ echo e(/** @var  string $foo */
```